### PR TITLE
Add to iOS App button on profile editor #4986

### PR DESCRIPTION
**Pull request by cklosowski**
#4986

### CODE REVIEW

File: includes/user-functions.php [hunk 1]
```diff
@@ -987,6 +991,22 @@ function edd_show_user_api_key_field( $user ) {
...
 			</tbody>
 		</table>
+		<table class="form-table">
+			<tbody>
+			<tr>
+				<th>
+					<?php printf( __( 'Easy Digital Downloads <a href="%s">iOS App</a>', 'easy-digital-downloads' ), 'https://itunes.apple.com/us/app/easy-digital-downloads/id625303275?ls=1&mt=8' ); ?>
+				</th>
+				<td>
+					<?php
```
Comment (pippinsplugins) [inline includes/user-functions.php]: Can we use wp_is_mobile() and only show this on iOS devices please?

Comment (cklosowski) [inline includes/user-functions.php]: :+1:

File: includes/user-functions.php [hunk 2]
```diff
@@ -987,6 +991,26 @@ function edd_show_user_api_key_field( $user ) {
...
+					$sitename = get_bloginfo( 'name' );
+					$ios_url  = 'edd://new?sitename=' . $sitename . '&siteurl=' . home_url() . '&key=' . $public_key . '&token=' . $token;
+					?>
+					<a class="button-secondary" href="<?php echo $ios_url; ?>"><?php _e( 'Add to iOS App', 'easy-digital-downloads' ); ?></a>
```
Comment (michaelbeil) [inline includes/user-functions.php]: Do we want `Connect` instead of `Add`?

Comment (cklosowski) [inline includes/user-functions.php]: @pippinsplugins preference here?

Comment (pippinsplugins) [inline includes/user-functions.php]: I like `Add`

**Pull request merged**